Recipes/TrainingDemo.recipe.json

```diff
@@ -65,6 +65,112 @@
           "Description": "The widgets in this layer are only displayed on the homepage."
         }
       ]
+    },
+    // Workflows
```

Comment:
There's a misunderstanding.

The Training Demo module is about demonstrating how you develop Orchard modules, not about how to utilize built-in user-level Orchard features. https://github.com/Lombiq/Orchard-Training-Demo-Module/issues/39 is about adding an example of a Workflows activity implementation, as simple as [`LogTask`](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Workflows/Activities/LogTask.cs), and (similar to all code in the module), with ample explanation of what it does.

Sorry if that wasn't clear from the admittedly short issue (though the context matters).

Reply:
I see, thanks But I think we can keep at least one of the workflows just for demonstration purposes. What do you think?
And we can have a second workflow that will represent the usage of the newly added activity. What do you think?